src/index.ts

```diff
@@ -1,7 +1,7 @@
 import { NativeModules } from 'react-native';
 
-import Camera from './Camera';
-import CameraScreen, { CameraType } from './CameraScreen';
+import Camera from './PlatformCamera';
+import CameraScreen, { CameraType } from './FCCCameraScreen';
```

Comment:
FCCCameraScreen? We can leave this in the CameraScreen folder and remove the class component